### PR TITLE
DOCS-5741 - site-specific messaging apm, synthetics, logs, security

### PR DESCRIPTION
**Pull request by cswatt**
### What does this PR do? What is the motivation?

Add site-specific messaging about feature availability for OKR5

### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

File: content/en/developers/_index.md [hunk 1]
```diff
@@ -33,6 +33,10 @@ If the solution you require is truly unavailable, you can contact [Datadog Suppo
 
 ### Partners and the Datadog Marketplace
 
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">Datadog Marketplace is not available for your selected <a href="/getting_started/site">Datadog site</a> ({{< region-param key="dd_site_name" >}}).</div>
```
Comment (anawishnoff) [inline content/en/developers/_index.md:37]: The Datadog Marketplace is now available in all sites (including Fed). Is this still needed?

**Review by drichards-87 — CHANGES_REQUESTED** [review]
Left you a couple of pieces of minor feedback + a question.

File: content/en/developers/_index.md (hunk 1 above)
Comment (cswatt) [reply content/en/developers/_index.md:37]: gotcha! removing marketplace fed messaging here

File: content/en/tracing/api_catalog/_index.md [hunk 2]
```diff
@@ -8,6 +8,10 @@ further_reading:
   text: "Datadog Service Catalog"
 ---
 
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">Dynamic Instrumentation is not supported for your selected <a href="/getting_started/site">Datadog site</a> ({{< region-param key="dd_site_name" >}}).</div>
```
[the remark this answers is not in the document]
Comment (cswatt) [reply content/en/tracing/api_catalog/_index.md:12]: nope—good catch, this was a copy/paste error (meant to say API Catalog)

**Review by drichards-87 — APPROVED** [review]

**Pull request merged**